### DISCUSSION TITLE
putListingsItem without issue but getListingsItem return issue

### DISCUSSION THREAD

**Discussion by dunghoangtekos** [amzn/selling-partner-api-models#3990] [General]
I've called putListingsItem many time for creating a listing and received response with empty issue array. But when I called getListingsItem, the response had issue code `8015` and unhelpful message "The data you provided for an attribute is invalid. Please refer the data definitions and resubmit with a valid value.". The categories was ["INVALID_ATTRIBUTE"]. 

This is the response from getListingItem:
`{
    "sku": "BOX-XR-PATN-1050",
    "summaries": [],
    "attributes": {
        "color": [
            {
                "value": "White",
                "language_tag": "en_US",
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "item_type_name": [
            {
                "value": "Color Laser Printer",
                "language_tag": "en_US",
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "item_package_weight": [
            {
                "unit": "kilograms",
                "value": 0.5,
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "manufacturer": [
            {
                "value": "Brother",
                "language_tag": "en_US",
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "model_name": [
            {
                "value": "Color Printer Toner TN-1050",
                "language_tag": "en_US",
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "bullet_point": [
            {
                "value": "Toner for Brother TN1050 TN-1050 Black Compatible",
                "language_tag": "en_US",
                "marketplace_id": "ATVPDKIKX0DER"
            },
            {
                "value": "Package Contents: 4-Pack TN760 black compatible toner cartridge",
                "language_tag": "en_US",
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "model_number": [
            {
                "value": "BOX-XR-PATN-1050",
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "product_description": [
            {
                "value": "The compatible toner for Brother TN-1050 has a print capacity of up to 1,000 pages in black and white at 5% coverage of the A4 sheet. The inks and pigments used for the production of compatible consumables are extremely high-performance and ensure perfect printing performance on paper until the last copy made. Long-lasting prints and ink mixes that do not damage the printer make the compatible Brother TN-1050 toner a toner equivalent in quality and performance to the originals of the brand, only more convenient in terms of cost.",
                "language_tag": "en_US",
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "supplier_declared_dg_hz_regulation": [
            {
                "value": "storage",
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "brand": [
            {
                "value": "SSC",
                "language_tag": "en_US",
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "mfg_series_number": [
            {
                "value": "TN1050",
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "country_of_origin": [
            {
                "value": "IT",
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "merchant_shipping_group": [
            {
                "value": "legacy-template-id",
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "item_name": [
            {
                "value": "Toner for Brother TN1050 TN-1050 Black Compatible",
                "language_tag": "en_US",
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "list_price": [
            {
                "currency": "USD",
                "value": 10.0,
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "batteries_required": [
            {
                "value": false,
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "item_type_keyword": [
            {
                "value": "Toner",
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "item_depth_width_height": [
            {
                "depth": {
                    "unit": "inches",
                    "value": 5.0
                },
                "width": {
                    "unit": "inches",
                    "value": 20.0
                },
                "height": {
                    "unit": "inches",
                    "value": 10.0
                },
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "merchant_suggested_asin": [
            {
                "value": "PRINTER001",
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "condition_type": [
            {
                "value": "new_new",
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "number_of_items": [
            {
                "value": 10,
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "item_package_dimensions": [
            {
                "length": {
                    "unit": "inches",
                    "value": 5.0
                },
                "width": {
                    "unit": "inches",
                    "value": 10.0
                },
                "height": {
                    "unit": "inches",
                    "value": 20.0
                },
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "size": [
            {
                "value": "5Dx10Wx20L",
                "language_tag": "en_US",
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "warranty_description": [
            {
                "value": "1 Year Manufacturer",
                "language_tag": "en_US",
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ],
        "part_number": [
            {
                "value": "BOX-XR-PATN-1050",
                "marketplace_id": "ATVPDKIKX0DER"
            }
        ]
    },
    "issues": [
        {
            "code": "8105",
            "message": "The data you provided for an attribute is invalid. Please refer the data definitions and resubmit with a valid value.",
            "severity": "ERROR",
            "categories": [
                "INVALID_ATTRIBUTE"
            ]
        }
    ],
    "offers": [],
    "fulfillmentAvailability": [],
    "procurement": []
}`

Is there anything I missed?

**Comment by shreeharsh-a** [comment]
@rugved1991

**Reply by dunghoangtekos** [reply]
@shreeharsh-a  I found that I could find the pushed listings anywhere in my Inventory pages, even in Draft product session. I can still using getListingsItem with SKU. The summaries in response was empty as well.

**Comment by chapmanjw** [comment]
@dunghoangtekos To troubleshoot this one, we will need to look at the underlying data for this listing. It's not abnormal for issues to be logged after submitting to the APIs. However, having an issue on an attribute that doesn't actually tell you which attribute is likely a bug on our end. Please contact developer support and provide the seller identifier + sku so that they can submit a ticket to our backend team to deep dive the issue: https://developer.amazonservices.com/support. Thanks!
